Ginger/Ginger/SolutionWindows/AddSolutionPage.xaml.cs

```diff
@@ -191,6 +191,28 @@ void AddDefaultDataSource()
             a.DSType = DataSourceBase.eDSType.MSAccess;
             RepositoryFolder<DataSourceBase> dsTargetFolder = WorkSpace.Instance.SolutionRepository.GetRepositoryItemRootFolder<DataSourceBase>();
             dsTargetFolder.AddRepositoryItem(a);
+
+           
+            // adding LiteDB while adding solution
+            byte[] litedbobj = Properties.Resources.LiteDB;
```

Comment:
Prefer not to use resources, we can put the file in folder and copy, then it is easier to replace and view